docs/BROWSER.md

```diff
@@ -45,6 +45,17 @@ For example,
 
 See [WIRING.md](WIRING.md) for details of where to put this.
 
+## Recording network interactions
+
+It is possible to record network interactions between the browser and the Jenkins instance.
+
+This works by setting up a proxy recording everything that goes through it and then configure the browser to use it.
+Supported drivers are: `firefox`, `chrome`, and `saucelabs-firefox`
+
+To achieve that, you'd run
+
+    HAR=true mvn install
```

Comment:
Please use more descriptive name for the config option. BTW, do we need to have this opt-in instead of on by default for supported drivers? Are there downsides of using the proxy?

Reply:
Changed to `RECORD_BROWSER_TRAFFIC`